### ISSUE TITLE
[Website] Link to Amazon AWS Marketplace from FAQ Page

### ISSUE THREAD

**Issue by nelsonic** [rethinkdb/rethinkdb#6097]
Hi ReThinkers!
This is _probably_ not the "_right_" place to post this question/issue. 
I could not _find_ a repo for the https://rethinkdb.com website on GitHub `else` I would post the issue there...

> **Context**: I'm _trying_ to convince "_Architects_" at `{Fortune_500_Company_Name_Here}` to use RethinkDB for an upcoming prototype ... but when one of them Clicked through the FAQ to the AWS Marketplace "launch" page we were _unpleasantly surprised_ which kinda killed the conversation...

On the https://rethinkdb.com/faq/ page 
![rethinkdb-fac-aws](https://cloud.githubusercontent.com/assets/194400/18384843/d367636a-7684-11e6-97b0-bfee61b2281c.png)
the link to `Amazon AWS` goes to: https://aws.amazon.com/marketplace/pp/B00E9EZ5DK
which has the following _unhelpful_ message:
![rethinkdb-aws-unavailable](https://cloud.githubusercontent.com/assets/194400/18384888/2b733476-7685-11e6-8c20-4cad551f7de9.png)

I did a bit more _searching_ on the website and found the https://rethinkdb.com/docs/paas/ page 
![rethinkdb-paas-page](https://cloud.githubusercontent.com/assets/194400/18384983/adaeef3e-7685-11e6-8520-3fada435ecd9.png)

which links to: https://aws.amazon.com/marketplace/pp/B013R60Q8Y which _appears_ to be the _latest_ version of RethinkDB ...
![rethinkdb-aws-marketplace](https://cloud.githubusercontent.com/assets/194400/18384988/b40c5998-7685-11e6-8b36-52bc9e4cb3aa.png)

I'm happy to "_fix_" it if there is a GitHub repo for your website...
Thanks!

**Comment by danielmewes** [comment]
Thanks @nelsonic, we indeed forgot to update that link when our listing changed (this was necessary when AWS switched to HVM virtualization instead of para-virtualized instances).

This particular page is in our "docs" repository. I just fixed the link here https://github.com/rethinkdb/docs/commit/d793ce97037c0c1a18212c786cde063f18147b7a . It will be pushed online with the next deploy of our website.
